src/pages/academy/videos/_all-videos.tsx

```diff
@@ -1,15 +1,19 @@
 import React, { useEffect, useState } from 'react'
-import PropTypes from 'prop-types'
 import { useQueryParam, StringParam } from 'use-query-params'
 import { StyledImg, Container, VideoGrid } from '../common/_styles'
 import VideoPlayer from '../components/_video-player'
 import VideoCard from './_video-card'
+import { VideoDataType } from './index'
 import { Flex } from 'components/containers'
 import { slugify } from 'common/utility'
 import { Text, LocalizedLinkText } from 'components/elements'
 import RightArrow from 'images/svg/tools/black-right-arrow.svg'
 
-const AllVideos = ({ video_data }) => {
+type AllVideosProps = {
+    video_data: VideoDataType
+}
+
+const AllVideos = ({ video_data }: AllVideosProps) => {
     const [show, setShow] = useState(false)
```

Comment:
Instead of 'false', should we use, ''; As overflow to 'false' doesn't sound right ?

Reply:
The `show` is actually meant to toggle on/off the video player. The overflow is just styled accordingly dependent on that state. I think this is fine.